packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx

```diff
@@ -155,10 +155,24 @@ export class AreaChartBasicExample extends React.Component<{}, IAreaChartBasicSt
 
     return (
       <>
-        <label>change Width:</label>
-        <input type="range" value={this.state.width} min={200} max={1000} onChange={this._onWidthChange} />
-        <label>change Height:</label>
-        <input type="range" value={this.state.height} min={200} max={1000} onChange={this._onHeightChange} />
+        <label for="changeWidth">change Width:</label>
+        <input
+          type="range"
+          value={this.state.width}
+          min={200}
+          max={1000}
+          id="changeWidth"
```

Comment:
Call this changeWidth/height _ Basic for AreaChart.Basic.Example and similarly  changeWidth/height _ Custom for AreaChart.CustomAccessibility etc

Reply:
Followed your inputs and fixed them.